projects/common/src/utilities/formatters/enum/display-string-enum.ts

```diff
@@ -1,13 +1,15 @@
 import { isNil, startCase } from 'lodash-es';
 
-export const displayStringEnum = (provided?: string): string => {
+export const displayStringEnum = (provided?: string, unknown: string = '-', separator: string = ' '): string => {
   if (isNil(provided)) {
-    return '-';
+    return unknown;
   }
 
   // This removes dashes and underscores and gives all words initial caps
   const startCased = startCase(provided);
 
+  const startCasedSeparated = startCased.replace(' ', separator);
+
   // We only want the first word capitalized.
-  return `${startCased[0]}${startCased.substr(1).toLowerCase()}`;
+  return `${startCasedSeparated[0]}${startCasedSeparated.substr(1).toLowerCase()}`;
```

Comment:
I feel like I commented on this at the time but who knows:
```suggestion
  return capitalize(startCasedSeparated);
```

Reply:
You did, but that doesn't work for certain use cases and I don't feel the alternative you provided is really any cleaner than this.

https://github.com/hypertrace/hypertrace-ui/pull/752/files#r609080471

Reply:
I'm confused - you said in that linked comment you agreed that `startCase(lowerCase(provided))` is cleaner, but it looks like it got merged without making that change. What use case isn't supported?

Here, I was just suggesting capitalize which is a 1:1 replacement of the explicit impl on line 14, but using
 ```ts 
 return capitalize(lowerCase(provided)).replace(' ', separator);`
 ```
 to replace lines 8-14 I think is even better.